Embedded System L1/AVR ATMEGA32/DIO/PROJECT/BIT_MATH.h

```diff
@@ -0,0 +1,16 @@
+/*********Author : Khalid Mosaad Zakaria */
+
+/*********Date   :   20/1/2021          */
+/*********Version: 1                     */
+/****************Desc: *******************/
+
+#ifndef _BIT_MATH_H
+#define _BIT_MATH_H
+
+#define SET_BIT(VAR,BITNO) (VAR) |=  (1 << (BITNO))
+#define CLR_BIT(VAR,BITNO) (VAR) &= ~(1 << (BITNO))
+#define TOG_BIT(VAR,BITNO) (VAR) ^=  (1 << (BITNO))
+#define GET_BIT(VAR,BITNO) (((VAR) >> (BITNO)) & 0x01)
+
+
+#endif
```

Comment:
-comments doesn't follow Sprints_CodeGuideLine of headers

![2](https://user-images.githubusercontent.com/44152298/127502084-967ab9b6-4fba-4be7-ba83-0af348f54c31.PNG)

[Sprints_CodeGuideLine.pdf](https://github.com/AhmedEssamMo/Sprints_Wave_9/files/6900596/Sprints_CodeGuideLine.pdf)